### PR TITLE
Fix fsharp patches

### PR DESCRIPTION
**Pull request by karajas**

### CODE REVIEW

**Comment by weshaggard** [comment]
@karajas are these complete?

File: targets/fsharp.props [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
+    <BuildRevision>171104</BuildRevision>
     <BuildCommand>$(ProjectDirectory)/src/buildfromsource.sh</BuildCommand>
     <NuGetConfigFile>$(ProjectDirectory)/NuGet.Config</NuGetConfigFile>
   </PropertyGroup>
 
   <ItemGroup>
     <EnvironmentVariables Include="BuildRevision=$(BuildRevision)" />
     <EnvironmentVariables Include="PATH=$(DotNetCliToolDir):$(PATH)" />
-    <RepositoryReference Include="core-setup" />
+<!--    <RepositoryReference Include="core-setup" />
```
Comment (weshaggard) [inline targets/fsharp.props]: Why are these commented out?

**Comment by karajas** [comment]
This may need the missing https://github.com/crummel/source-build-tarball-2.0.5/blob/master/patches/fsharp/0007-Don-t-restore-FSharp.Core-nupkg-use-source-version.patch to resolve the issues, will push an update.

**Comment by weshaggard** [comment]
Please push the patch and another commit which applies the patch to the source.

File: targets/fsharp.props (hunk 1 above)
Comment (crummel) [reply targets/fsharp.props]: Uncommented these.

**Comment by crummel** [comment]
Rebased on master, patches applied, patch 0007 restored in addition to Karthik's other changes.  Think this is ready to merge.

**Comment by weshaggard** [comment]
OK I'm going to merge and test it out.

**Pull request merged**